lib/codegen/src/print_errors.rs

```diff
@@ -152,6 +144,16 @@ fn pretty_preamble_error(
     Ok(())
 }
 
+/// prints ^~~~~~ verifier [ERROR BODY]
+fn print_error(w: &mut Write, indent: usize, s: String, err: VerifierError) -> fmt::Result {
+    write!(w, ";{1:0$}^", indent - 1, "")?;
```

Comment:
I don't think the indent is ever zero in the current verifier, so it's fine to not support that, but it'd be good to have an assert checking that.